### PR TITLE
gui(edit key modal): increase size of the closing icon

### LINKED ISSUE #1409: Increase the size of closing the popup icon ("X")
The closing icon of the key setup popup seems a bit small. We should possibly increase its size:

image

### PR DESCRIPTION
**Pull request by pythcoiner**
Fixes #1409 
image

### CODE REVIEW

**Comment by jp1ac4** [comment]
This new size looks better. The only thing is I think the empty space above "Set your key", which was already quite a lot, has increased. Perhaps the X should be in the same row as the title, or otherwise maybe there's a way to reduce some of the empty space.

**Comment by pythcoiner** [comment]
agree, i've tryed to limit the height of the cross button, but it makes the icon disapear

**Comment by pythcoiner** [comment]
i've moved the title in the same column:
image

**Comment by pythcoiner** [comment]
and centered widgets of the first row:
image

File: gui/src/installer/view/editor/mod.rs [hunk 1]
```diff
@@ -271,11 +271,14 @@ pub fn edit_key_modal<'a>(
             Column::new()
                 .spacing(25)
                 .push(Row::new()
+                    .push(Space::with_width(10))
```
Comment (jp1ac4) [inline gui/src/installer/view/editor/mod.rs:274]: The title and the "Select the signing device..." wording below previously had the same left-alignment. Would it be possible to remove this space in order to keep them aligned?

**Comment by pythcoiner** [comment]
i first add this space (before centering the title) as spacing from top & left were looking different, but now we are not so close from the top border, you're right it's looks better:
image

**Review by jp1ac4 — APPROVED** [review]
Tested ACK https://github.com/wizardsardine/liana/commit/f4596ab4b0974d0b3986c0431337b8a304333278.

Looks great :heart_eyes:

**Pull request merged**